src/util/string.h

```diff
@@ -473,6 +473,23 @@ inline void str_replace(std::string &str, const std::string &pattern,
 	}
 }
 
+ /**
++ * Replace all occurrences of \p pattern in \p str with \p replacement.
++ *
++ * @param str String to replace pattern with replacement within.
++ * @param pattern The pattern to replace.
++ * @param replacement What to replace the pattern with.
++ */
+inline void str_replace(std::wstring &str, const std::wstring &pattern,
+		const std::wstring &replacement)
+{
+	std::wstring::size_type start = str.find(pattern, 0);
+	while (start != str.npos) {
```

Comment:
str.npos looks weird
consider `std::wstring::npos` instead

Reply:
I am giving the change to MoNTE48, and changes to the instances that were already there as well.

Reply:
This is copied and pasted from the existing function--the standard library probably puts npos in the object just for that reason (it can avoid copy&paste errors when people make future changes as well). The file uses `str.npos` throughout, with or without the PR.